cli_tools/common/disk/inspect.go

```diff
@@ -63,50 +71,197 @@ func NewInspector(wfAttributes daisycommon.WorkflowAttributes, network string, s
 	daisycommon.SetWorkflowAttributes(wf, wfAttributes)
 	wf.Vars["network"] = daisy.Var{Value: network}
 	wf.Vars["subnet"] = daisy.Var{Value: subnet}
-	return &defaultInspector{wf}, nil
+	return &bootInspector{[]string{}, &defaultDaisyWorker{wf}}, nil
 }
 
-// defaultInspector implements disk.Inspector using a Daisy workflow.
-type defaultInspector struct {
-	wf *daisy.Workflow
+// bootInspect implements disk.Inspector using the Python boot-inspect package,
+// executed on a worker VM using Daisy.
+type bootInspector struct {
+	traceLogs []string
+	worker    daisyWorker
+}
+
+// daisyWorker is a facade over daisy.Workflow to facilitate mocking.
+type daisyWorker interface {
```

Comment:
Consider renaming. Since this abstracts daisy.Workflow and is not exported, using daisyWorkflow should be ok.